examples/small.md

```diff
@@ -5,43 +5,54 @@ author: test author
 
 # This is a small example
 
-```{python}
+This example is not a valid Stitch markdown document. It only contains some Stitch code examples.
```

Comment:
Yes. The original version runs fine. Examples changes are optional.

I think the examples would be primarily viewed from GitHub. And my changes to examples improve readability when viewing from GitHub. But with these changes the file is no longer a valid Stitch document.